pkg/registry/core/componentstatus/validator.go

```diff
@@ -42,7 +45,11 @@ type Server struct {
 	Port        int
 	Path        string
 	EnableHTTPS bool
+	CertFile    string
+	KeyFile     string
```

Comment:
probably want this to be the certificate object, not require loading files on disk here